pythoncalculator/divide.py

```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
```

Comment:
```suggestion
def divide(x, y):
```
Hi Tara, I'm going to commit this change - making sure the variables are consistent